wpilibc/src/main/native/include/frc/Tracer.h

```diff
@@ -67,6 +84,14 @@ class Tracer {
 
   hal::fpga_clock::time_point m_startTime;
   hal::fpga_clock::time_point m_lastEpochsPrintTime = hal::fpga_clock::epoch();
+  bool m_publishNT;
```

Comment:
```suggestion
  bool m_publishNT = false;
```
in c++ primitive types don't have a default value, this is what is triggering the sanitizers